apps/src/templates/projects/ProjectsPromo.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import {TwoColumnActionBlock} from '@cdo/apps/templates/studioHomepages/TwoColumnActionBlock';
+import {pegasus} from '@cdo/apps/lib/util/urlHelpers';
+import i18n from '@cdo/locale';
+import color from '@cdo/apps/util/color';
+
+const ProjectsPromo = () => {
+  return (
+    <TwoColumnActionBlock
+      imageUrl={pegasus('/images/athome/fill-970x562/app-lab.png')}
+      subHeading={i18n.projectPromoHeading()}
+      description={i18n.projectPromoDescription()}
+      backgroundColor={color.purple}
```

Comment:
I changed the background color to be more accessible. White on light blue has awful contrast, especially for low-vision users.

**BEFORE**
<img width="1113" alt="Screen Shot 2021-11-04 at 4 42 37 PM" src="https://user-images.githubusercontent.com/12300669/140444713-1078feb5-63aa-49ad-b2c8-667de02938a6.png">

**AFTER**
<img width="1183" alt="Screen Shot 2021-11-04 at 4 43 19 PM" src="https://user-images.githubusercontent.com/12300669/140444730-7a65365e-6d98-413b-806e-33aa74bae6dd.png">

@markabarrett How do you feel about this? Are there alternatives you'd like to try?